### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by Ashah15**

### CODE REVIEW

**Comment by Ispirett** [comment]
Hey! folks really great job with this project it looks awesome :1st_place_medal: . There are just a few things to address. 
- [ ] You are missing the description " 
One account for everything ", it should be under the title
- [ ] For the icons you should use separate images and text instead of one image for everything.
- [ ] Image tags should have an 'alt' attribute it should have a descriptive name for the image, we use that for screen readers and if the image is unavailable.
- [ ] Remove the " aria-described" you don't need it for this project. It's also not pointing any element in your page. 
- [ ] Bonus tip if you can carry the sign-up button and text a little more to the right it will just be perfect :+1: 
Nevertheless, you folks are doing great!! Please make changes and resubmit for code review.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # FORMS
 HTML forms
+## the live site is at https://ashah15.github.io/FORMS/
+
+### Description: 
+Created a sign up form 
+
+#### Authors:
+[Asha] (https://github.com/Ashah15)
+[Micheals] (https://github.com/MarvellousUbani)
```
Comment (bolah2009) [inline README.md]: `AWESOME`

Nice job creating a `README.md` file, this gives the project more description of what it does and how it does it.

-----

`SUGGESTION`

- Add the assignment link, the project specification, screenshots and other pieces of information that would describe the project.
- When linking a file using `markdown` don't put space between the square brackets and the parenthesis eg. 
`[Asha](https://github.com/Ashah15)` would appear as [Asha](https://github.com/Ashah15) you can learn more about markdown syntax [here](https://www.markdownguide.org/cheat-sheet/)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+			<div class="col-4"><span class="smallFont float-right">Already have an Intuit account?<br>
+              <a href="#">I forgot my User ID or Password</a></span>
+			<span class="float-right"><button class="btn btn-default btn-blue smallFont"><i class="fa fa-lock" ></i> Sign In</button></span>
+		</div>
+		</div>
+
+		<div class="row text-center imgSide my-3">
+			<div class="col">
+			<p class="text-center"><img class="img-fluid" src="img/logo.png"/ alt="website logo"></p>
+			<p class="text-center"><img class="img-fluid" src="img/mint.png"/ alt="mint logo"><img class="img-fluid" src="img/quickbooks.png" alt="quickbooks logo" /><img class="img-fluid" src="img/turbotax.png" alt="turbotax logo" /></p>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

Consider removing the `/` after `src="img/mint.png"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+			<div class="col-8"></div>
+			<div class="col-4"><span class="smallFont float-right">Already have an Intuit account?<br>
+              <a href="#">I forgot my User ID or Password</a></span>
+			<span class="float-right"><button class="btn btn-default btn-blue smallFont"><i class="fa fa-lock" ></i> Sign In</button></span>
+		</div>
+		</div>
+
+		<div class="row text-center imgSide my-3">
+			<div class="col">
+			<p class="text-center"><img class="img-fluid" src="img/logo.png"/ alt="website logo"></p>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

Consider removing the `/` after `src="img/logo.png""`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+			<div class="white-box">
+              <div class="formtitlearea">
+				<h4 class="text-center">Create an Intuit account</h4>
+				<p class="text-center formsubtitle">One account for everything Intuit, including Mint. <a href="#">Learn more</a></p>
+				</div>
+        <div class="formarea">
+          <form action="myform.cgi">
+          	<div class="form-group">
+			    <label for="emailaddress">Email address</label>
+			    <input type="email" class="form-control" id="emailaddress" name="email">
```
Comment (bolah2009) [inline index.html:47]: `AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+			<label for="phone">Phone (Recommended):</label>
+			<input type="tel" id="phone" class="form-control" name="phone"
+			       pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+			       required>
+			       <small>Standardcalls,messaging or data rates may apply</small>
+			 </div>
+            
+              <div class="form-group">
+			  Password:
+			  <input type="password" name="psw" class="form-control">
```
Comment (bolah2009) [inline index.html:61]: `SUGGESTION`

Check comment above on associating `<input>` and `<label>` element.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+				<div class="tocarea smallFont">
+					Invisible reCAPTCHA by <a href="#">Google Privacy Policy</a> and Terms of Use.
+				</div>
+			</div>
+           </div>
+		</div>
+
+		<div class="row">
+			<div class="col footer">
```
Comment (bolah2009) [inline index.html:82]: `SUGGESTION`

Make this a footer element to make it have more semantic meaning.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+<!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+
+  </body>
+  </html>
```
Comment (bolah2009) [inline index.html:99]: `SUGGESTION`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE/code Editor  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

Screenshot of how to format code using VS Code

**Review by bolah2009 — APPROVED** [review]
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

I'm impressed and proud of your progress, you should be too :clap:

### Important: I strongly recommend following the suggestions in the review comments and also make your sign in button look like the picture below before you merge.

### Current

currentsignin

### Preffered 

INTUITSIGNIN

Good luck with your next project
Keep it up :star2:

Cheers
Bola

**Pull request merged**